bundles/org.openhab.binding.wlanthermo/README.md

```diff
@@ -76,14 +76,14 @@ If username/password is given in the thing, most channels are writeable.
 
 All channels are read only!
 
-#### The device itself provides the following channels:
+#### The device itself provides the following channels
```

Comment:
Same chapter name as in line 40. Not any different than before, but seeing that you rephrased some other chapter tiles, just wanted to mention. 🙂

Reply:
Didn't promise 100% coverage ;-) Tried to fix some, but also noticed that more readme files need extra attention as they seem old and the template format has been changed over time while older ones where not adapted. I really like the recent ones with tables, migrating takes time. Might come back to this.